pkg/cmd/server/server.go

```diff
@@ -197,6 +207,8 @@ func newServer(namespace, baseName string, config serverConfig, logger *logrus.L
 	if err != nil {
 		return nil, err
 	}
+	clientConfig.QPS = config.clientQPS
```

Comment:
I don't know why I didn't think of this sooner - should we validate these are positive numbers?

Reply:
```
	if config.clientQPS <= 0 {
		return errors.New("client-qps can be only a positive number")
	}
```
Something like this? Maybe some better wording, not a native english speaker:)

Reply:
That's good, or `client-qps must be positive`